### DISCUSSION TITLE
The connection for this site is not secure

### DISCUSSION THREAD

**Discussion by steven86m** [postalserver/postal#2894] [Help with using Postal]
I'm getting this message when trying to connect to my web ui

"The connection for this site is not secure"

I've tried the IP as well as the FQDN, Please help.

I also have ERR_SSL_PROTOCOL_ERROR

**Comment by willpower232** [comment]
Did you use nginx or caddy or something else when setting up your Postal server?

**Reply by steven86m** [reply]
I used the Caddy script from the install guide

**Reply by steven86m** [reply]
docker run -d \
   --name postal-caddy \
   --restart always \
   --network host \
   -v /opt/postal/config/Caddyfile:/etc/caddy/Caddyfile \
   -v /opt/postal/caddy-data:/data \
   caddy

**Reply by willpower232** [reply]
are you able to see any issues with `docker logs postal-caddy`?
